gdk4/src/auto/popup_layout.rs

```diff
@@ -6,6 +6,9 @@ use crate::AnchorHints;
 use crate::Gravity;
 use crate::Rectangle;
 use glib::translate::*;
+#[cfg(any(feature = "v4_2", feature = "dox"))]
```

Comment:
You'll want to set an override in `Cargo.toml` so that `v4_2` does not check for >=4.2 via pkg-config but only for `>=4.1.2`.